js/renovation/ui/grids/data_grid_light/data_grid_light.tsx

```diff
@@ -26,18 +33,60 @@ export const viewFunction = (viewModel: DataGridLight): JSX.Element => (
   >
     <div className="dx-datagrid dx-gridbase-container" role="grid" aria-label="Data grid">
       <TableHeader columns={viewModel.props.columns} />
-      <TableContent columns={viewModel.props.columns} dataSource={viewModel.props.dataSource} />
+      <TableContent columns={viewModel.props.columns} dataSource={viewModel.visibleItems} />
+      <GridPager
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...viewModel.props.pager}
+        pageCount={viewModel.pagingPageCount}
+        totalCount={viewModel.props.dataSource.length}
+
+        pageIndex={viewModel.pagingPageIndex}
+        pageIndexChange={viewModel.onPageIndexChange}
+
+        pageSize={viewModel.pagingPageSize}
+        pageSizeChange={viewModel.onPageSizeChange}
+      />
     </div>
   </Widget>
 );
 
+@ComponentBindings()
+export class PagingProps {
+  @OneWay()
+  enabled = true;
+
+  @OneWay()
+  pageIndex = 0;
+
+  @OneWay()
+  pageSize: number | 'all' = 20;
```

Comment:
In the old DataGrid if select 'All' item, pageSize is assigned to 0. And user should assign pageSize to 0 to select 'all item'.
https://codepen.io/vconst/pen/dyVXbQK
I think we should implement the same logic.